lib/pltr/v2/actions/books.js

```diff
@@ -8,8 +8,8 @@ import {
 } from '../constants/ActionTypes'
 import { book } from '../store/initialState'
 
-export function addBook() {
-  return { type: ADD_BOOK, book }
+export function addBook(newBookId, attributes) {
```

Comment:
As discussed before, please don't supply the `newBookId` and spread `attributes` into four arguments.